actions/contacts/create.go

```diff
@@ -23,13 +25,19 @@ import (
 // @Security	bux-auth-xpub
 func (a *Action) create(c *gin.Context) {
 
-	fullName := c.GetString("full_name")
-	paymail := c.GetString("paymail")
-	pubKey := c.GetString("pubKey")
+	var contactStruct engine.Contact
```

Comment:
Consider another name without "Struct" suffix.

Reply:
Changed

Reply:
I can't see that you've changed it :sweat_smile:

Reply:
Changed in another file LoL